enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -298,6 +300,37 @@ def get_policy_class_by_type(cls, policy_type):
                 return policy_class
         return None
 
+    @property
+    def get_all_policies_associated_to_subsidy(self):
+        """
+        Retrives all policies associated to instance's subsidy
+        """
+        enterprise_customer_uuid = str(self.enterprise_customer_uuid)
+        subsidy_uuid = str(self.subsidy_uuid)
+        return SubsidyAccessPolicy.objects.filter(
+            enterprise_customer_uuid=enterprise_customer_uuid,
+            subsidy_uuid=subsidy_uuid,
+        )
+
+    def total_spend_limit_for_all_policies_associated_to_subsidy(self, policies):
+        """
+        Sums the policies spend_limit excluding the db's instance of this policy
+        """
+        policy_balances = []
+        for policy in policies:
+            policy_uuid = getattr(policy, 'uuid')
+            if policy_uuid != self.uuid:
+                spend_available_usd_cents = getattr(policy, 'spend_limit')
+                policy_balances.append(spend_available_usd_cents)
+        policy_balances.append(self.spend_limit)
+        return sum(policy_balances)
+    
+    @property
+    def is_spend_limit_updated(self):
+        if SubsidyAccessPolicy.objects.filter(uuid=self.uuid).first():
```

Comment:
This conditional is specifically added to make the feature testable without it affecting every existing test.

Without employing this conditional, test related to the `SubsidyAccessPolicy` would need to be updated.
(Previous CI run: https://github.com/openedx/enterprise-access/actions/runs/9368112719/job/25789314544)
The conditional has no effect on the initial intention of this ticket which is to validate on changes to `spend_limit`